op-chain-ops/script/script.go

```diff
@@ -365,6 +397,47 @@ func (h *Host) HasPrecompileOverride(addr common.Address) bool {
 	return ok
 }
 
+// onEnter is a trace-hook, which we use to apply changes to the state-DB, to simulate isolated broadcast calls,
+// for better gas estimation of the exact broadcast call execution.
+func (h *Host) onEnter(depth int, typ byte, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
+	if len(h.callStack) == 0 {
+		return
+	}
+	parentCallFrame := h.callStack[len(h.callStack)-1]
+	if parentCallFrame.Prank == nil {
+		return
+	}
+
+	if parentCallFrame.Prank.Broadcast {
+		if vm.OpCode(typ) == vm.CALL && to != VMAddr {
+			sender := parentCallFrame.Ctx.Caller()
```

Comment:
```suggestion
			sender := parentCallFrame.Ctx.Address()
```
The address of the parent = sender of the current.

The prank is set up in the parent, since it is prepared and then applied later, and might repeat multiple sub-calls, hence the difference with the `prank.sender`.